README.md

```diff
@@ -257,12 +257,28 @@ A value that implements the Traversable specification must also
 implement the Functor specification.
 
 1. `t(u.sequence(f.of))` is equivalent to `u.map(t).sequence(g.of)`
-where `t` is a natural transformation from `f` to `g` (naturality)
-
-2. `u.map(x => Id(x)).sequence(Id.of)` is equivalent to `Id.of(u)` (identity)
-
-3. `u.map(Compose).sequence(Compose.of)` is equivalent to
-   `Compose(u.sequence(f.of).map(x => x.sequence(g.of)))` (composition)
+for any `t` such that `t(x).map(a)` is equivalent to `t(x.map(a))` (naturality)
```

Comment:
It's not clear here what `x` is. Maybe we should rewrite it as `t(f.of(x)).map(a) ≡ t(f.of(x).map(a))`?